frontend/src/components/core/Timeline.tsx

```diff
@@ -0,0 +1,60 @@
+import { Icon } from "@blueprintjs/core"
+import { BlueprintIcons_16Id } from "@blueprintjs/icons/src/generated/16px/blueprint-icons-16.ts"
+import Pill from "./Pill"
+
+interface metadata {
+    icon?: BlueprintIcons_16Id,
+    value: string,
+}
+
+export interface timelineEvent {
```

Comment:
Not: exported types generally should be capitalized

Reply:
addressed